client/state/reader/start/reducer.js

```diff
@@ -62,7 +76,21 @@ export function isRequestingRecommendations( state = false, action ) {
 	return state;
 }
 
+export function recommendationsInteractedWith( state = [], action ) {
```

Comment:
Not in the UI, but it's pretty useful for debugging at the moment :) I'll leave it as an array for now.